stl/inc/ranges

```diff
@@ -8427,6 +8429,8 @@ namespace ranges {
 
     template <class _Container>
     struct _To_class_fn {
+        _STL_INTERNAL_STATIC_ASSERT(!is_const_v<_Container>, "the target type must be cv-unqualified");

```

Comment:
you don't need the string there.

Reply:
Out of curiosity, why put the string in the other places and not here?

Reply:
`_STL_INTERNAL_STATIC_ASSERT` is for us and `static_assert` for users.
If user can trigger `_STL_INTERNAL_STATIC_ASSERT` then our implementation has a bug.
We can trigger `_STL_INTERNAL_STATIC_ASSERT` while refactoring some code or maybe adding a new feature.